horovod/mxnet/tensor_util.cc

```diff
@@ -133,15 +133,6 @@ NDArray* TensorUtil::New(int device, int dtype) {
 
 void TensorUtil::Free(NDArray* tensor) { delete tensor; }
 
-// Resize tensor to nDimension with length size[i] in dimension i
-void TensorUtil::ResizeNd(NDArray* tensor, int nDimension, int64_t* size) {
```

Comment:
Maybe just keep the helper function, change the signature if needed, and implement it using `MXNDArrayReshape64`?

Reply:
sounds good.